src/encode_task_subsample_ctl.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+import sys
+import os
+import argparse
+from encode_lib_common import (
+    assert_file_not_empty, get_num_lines, log, ls_l, mkdir_p, rm_f,
+    run_shell_cmd, strip_ext_ta)
```

Comment:
Would like to see one import per line, makes future diffs nicer:
```suggestion
from encode_lib_common import (
    assert_file_not_empty,
    get_num_lines,
    log,
    ls_l,
    mkdir_p,
    rm_f,
    run_shell_cmd,
    strip_ext_ta,
)
```